src/main/java/Duke.java

```diff
@@ -1,10 +1,147 @@
+            break;
+        case COMMAND_BYE:
+            exitProgram();
+            break;
+        default:
+            addTask(inCommand);
+            break;
+        }
+    }
+
+    private static void exitProgram() {
+        System.out.println("Bye. Hope to see you again soon!");
+        System.exit(0);
+    }
+
+    private static void addTask(String inCommand) {
+        String[] sentenceArr = inCommand.split(" ");
+        String[] taskToInput;
+        String printTask;
+        switch (sentenceArr[0]) {
+        case COMMAND_TODO:
+            taskToInput = Arrays.copyOfRange(sentenceArr, 1, sentenceArr.length);
+            printTask = arrayToString(taskToInput);
+            taskItems[itemNum] = new Task(printTask);
+            taskItems[itemNum].taskType = 'T';
+            System.out.println("Got it. I've added this task:\n" + "[T][ ] " + printTask +
+                    "\nNow you have " + (itemNum + 1) +  " tasks in the list.");
+            itemNum++;
+            break;
+        case COMMAND_DEADLINE:
+            int getDateIndex = Arrays.asList(sentenceArr).indexOf("/by");
+            taskToInput = Arrays.copyOfRange(sentenceArr, 1, getDateIndex);
+            String[] date = Arrays.copyOfRange(sentenceArr, getDateIndex + 1, sentenceArr.length);
+            String[] finalDeadlineArr = concatArray(taskToInput, date, " (by:");
+            printTask = arrayToString(finalDeadlineArr);
+            taskItems[itemNum] = new Task(printTask);
+            taskItems[itemNum].taskType = 'D';
+            System.out.println("Got it. I've added this task:\n" + "[D][ ] " + printTask +
+                    "\nNow you have " + (itemNum + 1) +  " tasks in the list.");
+            itemNum++;
+            break;
+        case COMMAND_EVENT:
+            int getTimeIndex = Arrays.asList(sentenceArr).indexOf("/at");
+            taskToInput = Arrays.copyOfRange(sentenceArr, 1, getTimeIndex);
+            String[] timing = Arrays.copyOfRange(sentenceArr, getTimeIndex + 1, sentenceArr.length);
+            String[] finalEventArr = concatArray(taskToInput, timing, "(at:");
+            printTask = arrayToString(finalEventArr);
+            taskItems[itemNum] = new Task(printTask);
+            taskItems[itemNum].taskType = 'D';
+            System.out.println("Got it. I've added this task:\n" + "[E][ ] " + printTask +
+                    "\nNow you have " + (itemNum + 1) +  " tasks in the list.");
+            itemNum++;
+            break;
+        default:
+            System.out.println("Invalid input");
+            break;
+        }
+
+    }
+
+    private static String[] concatArray(String[] taskToInput, String[] time, String timeFormat) {
+        int length = taskToInput.length + time.length + 2;
+        String[] finalTask = new String[length];
+        int pos = 0;
```

Comment:
Try specifying which position this int is referring to? Looking at the code I'm not quite sure this pos is referring to what